salt/metalk8s/container-engine/containerd/files/50-metalk8s.conf.j2

```diff
@@ -1,4 +1,7 @@
 [Service]
+# Allow passing arguments to containerd daemon
+ExecStart=
+ExecStart=/usr/bin/containerd {{ args | join(' ') }}
```

Comment:
Not really resilient if there is space in one of the args... but I assume it'll not be the case, at least it's not for now.

Reply:
I can trick it like this:

```jinja
/usr/bin/containerd "{{ args | join('" "'}}"
```

Reply:
Done